drivers/flash/spi_nor_gd25q64c.c

```diff
@@ -146,6 +160,36 @@ int gd25q64c_lock_otp_register(const struct device *dev, uint8_t reg_idx)
 	ret = spi_nor_cmd_write_data(dev, GD25Q64C_CMD_WRSR2, &reg, 1);
 
 	spi_nor_wait_until_ready(dev);
+
+out:
```

Comment:
`out` again :P Can we actually say if it's a failure or whatever it is?

Reply:
it's just out, not matter how we got there, due to error or it's normal execution flow.

Reply:
`release_resource` ? `release_spi`? `cleanup`?